src/Web/ClientApp/src/app/components/helpers/success/success.component.html

```diff
@@ -0,0 +1,9 @@
+<article *ngIf="visible" class="message is-success">
+    <div class="message-header">
+        <p>Success</p>
+    </div>
+    <div class="message-body">
+        Environment Variables Saved
+    </div>
+    <progress class="progress is-success" [attr.value]="counter" max="3"></progress>
```

Comment:
You could make the max 3000 and increment every milisecond so the progress bar would look fluid